src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -224,6 +224,12 @@ trait WorkflowComponent {
       findWorkflowsBySubmissionId(submissionId).filter(_.status inSet(statuses.map(_.toString))).result
     }
 
+    def countWorkflowsByQueueStatus: ReadAction[Map[String, Int]] = {
+      findQueuedAndRunningWorkflows.result.map { recs =>
+        recs.groupBy(_.status).mapValues(_.length)
```

Comment:
no thumb yet... this is doing the group and count scala side, database would be better

findQueuedAndRunningWorkflows.groupBy(_.status).length
maybe